<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schedule::ScheduleDialog</class>
 <widget class="QDialog" name="schedule::ScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0,0">
   <item row="0" column="0">
    <widget class="QComboBox" name="dates_box"/>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="add_btn">
     <property name="icon">
      <iconset resource="../schedule.qrc">
       <normaloff>:/schedule/add.svg.p</normaloff>:/schedule/add.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="del_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="icon">
      <iconset resource="../schedule.qrc">
       <normaloff>:/schedule/remove.svg.p</normaloff>:/schedule/remove.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="edit_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="icon">
      <iconset resource="../schedule.qrc">
       <normaloff>:/schedule/edit.svg.p</normaloff>:/schedule/edit.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTableView" name="tasks_view">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="settings_btn">
     <property name="icon">
      <iconset resource="../schedule.qrc">
       <normaloff>:/schedule/settings.svg.p</normaloff>:/schedule/settings.svg.p</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../schedule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>schedule::ScheduleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>schedule::ScheduleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
